src/mds/MDSMap.cc

```diff
@@ -150,8 +150,9 @@ void MDSMap::dump(Formatter *f) const
   f->dump_int("root", root);
   f->dump_int("session_timeout", session_timeout);
   f->dump_int("session_autoclose", session_autoclose);
-  f->dump_stream("min_compat_client") << to_integer<int>(min_compat_client) << " ("
-				      << min_compat_client << ")";
+  f->open_object_section("required_client_features");
```

Comment:
this breaks the dashboard tests. might want to update `__mdsmap_schema` in `qa/tasks/mgr/dashboard/test_health.py` accordingly.

Reply:
Thanks for pointing that out @tchaikov !